### DISCUSSION TITLE
Is there any way to get voice channel audio?

### DISCUSSION THREAD

**Discussion by JMoreno97** [Rapptz/discord.py#6050] [Q&A] [answered]
Hello, I was thinking on doing a bot that could interact with us by voice (he would receive the commands by voice) so I need to allow the bot to connect a voice channel and get the audio we are talking. Is there any way to to that?

Thanks.

**Comment by ShineyDev** [comment, marked as answer]
See #1094.

**Reply by JMoreno97** [reply]
Thanks!
